app/src/main/res/values-land/styles.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <style name="Heading1" parent="TextView.Common1">
```

Comment:
I'm not sure that we actually want to change this for all landscape headings. Why is this needed?

FWIW I think we can just use 'Heading1ViewStart' since that's the one we should always be using for non-centered headings.